test/selenium/selenium-machines-basic.py

```diff
@@ -127,14 +127,14 @@ def testVmStatus(self):
             "fails too often, https://github.com/cockpit-project/cockpit/issues/13072")
     def testCreateVMWithISO(self):
         name = 'test_iso'
-        iso = '/home/{}.iso'.format(name + str(time.time()).split('.')[0])
```

Comment:
This is not related to the changes that are implied in the commit message title.